Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -14,6 +14,15 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
         return textField
     }()
 
+    lazy var appSwitchStageToggleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "App Switch Stage"
```

Comment:
Nit, wording to make this more clear
```suggestion
        label.text = "App Switch Staging Env"
```

Reply:
take or leave